test/metabase/lib/expression_test.cljc

```diff
@@ -414,7 +416,9 @@
       (is (= "newly-named-expression"
              (lib/display-name query expr)))
       (is (not= (lib.options/uuid orig-expr)
-                (lib.options/uuid expr))))
+                (lib.options/uuid expr)))
+      (is (= (:ident (lib.options/options orig-expr))
```

Comment:
nit: assuming ident are going to be read in many places, could be nice to have a `lib.options/ident` accessor, similar to `lib.options/uuid` above

Reply:
I see now that `lib.options/ident` is already added in the [next PR in the stack](https://github.com/metabase/metabase/pull/50085).